sites/automationworld/redirect-handler.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  check: (from) => {
+    // Digital Edtion
+    let pattern = /\/sites\/default\/files\/digital_edition\/(.*)/;
+    let matches = pattern.exec(from);
+    if (matches) {
+      return { to: `https://digitaleditions.automationworld.com/${matches[1]}` };
+    }
+
+    // S3 Redirects
+    pattern = /\/sites\/default\/files\/(.*)/;
```

Comment:
Make each of these their own function so you do not need to use `let` and var reassignment. Plus, if you had unit tests, then you could test each function separately. :)